src/plugins/auto_batch/src/plugin.cpp

```diff
@@ -82,9 +82,14 @@ ov::Any Plugin::get_property(const std::string& name, const ov::AnyMap& argument
             return {it->second};
         }
     } else if (name == ov::supported_properties.name()) {
-        return std::vector<ov::PropertyName>{
-            ov::PropertyName{ov::supported_properties.name(), ov::PropertyMutability::RO},
-            ov::PropertyName{ov::device::full_name.name(), ov::PropertyMutability::RO}};
+        // return supported_configKeys;
+        std::vector<ov::PropertyName> property_name;
+        property_name.push_back(ov::PropertyName{ov::supported_properties.name(), ov::PropertyMutability::RO});
+        property_name.push_back(ov::PropertyName{ov::device::full_name.name(), ov::PropertyMutability::RO});
+        for (auto& it : supported_configKeys) {
+            property_name.push_back(ov::PropertyName{it, ov::PropertyMutability::RO});
```

Comment:
We need to add  RW/RO to supported_configKeys, I think the original intention that config means RW.